src/components/AddressBookContainer/AddressBookContainer.tsx

```diff
@@ -0,0 +1,21 @@
+import { AddressBook, BackArrow } from "@govtechsg/tradetrust-ui-components";
+import { usePersistedConfigFile } from "./../../common/hook/usePersistedConfigFile";
+import { Link } from "react-router-dom";
+import React, { FunctionComponent } from "react";
+import { NavigationBar } from "../NavigationBar";
+
+export const AddressBookContainer: FunctionComponent = () => {
+  const { configFile } = usePersistedConfigFile();
+  return (
+    <>
+      <NavigationBar />
```

Comment:
Do u think the nav bar can be in the router level?

Reply:
i think i tried b4 like cannot due to how this project was been setup (the part on various form routes logic), i can try again see how let u know again

Reply:
ok looks like can 👍